### PR TITLE
Prac 08 feedback

### PR DESCRIPTION
**Pull request by JevanChandra**
Please help review my code
@AdriantoHartono
@MikeLeongZhiFai
@Riston-gitting-gud
@a-rosa

### CODE REVIEW

File: cp1404practicals/prac_08/box_layout.kv
Comment (AdriantoHartono) [inline cp1404practicals/prac_08/box_layout.kv:1]: Nice code 👌

File: cp1404practicals/prac_08/box_layout_demo.py
Comment (AdriantoHartono) [inline cp1404practicals/prac_08/box_layout_demo.py:1]: Great code, and already made neatly👍

File: cp1404practicals/prac_08/convert_miles_km.kv
Comment (AdriantoHartono) [inline cp1404practicals/prac_08/convert_miles_km.kv:1]: Great code!

File: cp1404practicals/prac_08/convert_miles_km.py
Comment (AdriantoHartono) [inline cp1404practicals/prac_08/convert_miles_km.py:1]: Nice code 🙏

File: cp1404practicals/prac_08/dynamic_labels.kv
Comment (AdriantoHartono) [inline cp1404practicals/prac_08/dynamic_labels.kv:1]: Great 👌

File: cp1404practicals/prac_08/dynamic_labels.py
Comment (AdriantoHartono) [inline cp1404practicals/prac_08/dynamic_labels.py:1]: Nice code 👍

File: cp1404practicals/prac_08/squaring.kv
Comment (AdriantoHartono) [inline cp1404practicals/prac_08/squaring.kv:1]: Nice code 👌

File: cp1404practicals/prac_08/squaring.py
Comment (AdriantoHartono) [inline cp1404practicals/prac_08/squaring.py:1]: Great👍

File: cp1404practicals/prac_08/convert_miles_km.kv [hunk 1]
```diff
@@ -0,0 +1,32 @@
+BoxLayout:
```
Comment (Riston-gitting-gud) [inline cp1404practicals/prac_08/convert_miles_km.kv:1]: Nice code , it works!

File: cp1404practicals/prac_08/convert_miles_km.py
Comment (MikeLeongZhiFai) [reply cp1404practicals/prac_08/convert_miles_km.py:1]: Great code! Worked well!

File: cp1404practicals/prac_08/box_layout.kv [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+        on_press: app.clear()
+    TextInput:
+        id: input_name
+        text: ''
+    Button:
+        text: 'Greet'
+        on_press: app.handle_greet()
+    Label:
+        id: output_label
+        text: 'Jevan Chandra'
```
Comment (a-rosa) [inline cp1404practicals/prac_08/box_layout.kv:16]: I think the initial text in the label should be empty

File: cp1404practicals/prac_08/convert_miles_km.kv [hunk 3]
```diff
@@ -0,0 +1,32 @@
...
+                font_size: 40
+                on_press: app.handle_increment(1)
+            Button:
+                text: 'Down'
+                font_size: 40
+                on_press: app.handle_increment(-1)
+#    Button:
+#        text: 'Convert miles to km'
+#        font_size: 40
+#        on_press: app.handle_calculate()
```
Comment (a-rosa) [inline cp1404practicals/prac_08/convert_miles_km.kv:26]: I think the button should be shown in the window

**Pull request merged**